src/libs/Network/NetworkStore.ts

```diff
@@ -1,32 +1,28 @@
-import lodashGet from 'lodash/get';
 import Onyx from 'react-native-onyx';
-import _ from 'underscore';
 import ONYXKEYS from '../../ONYXKEYS';
+import Credentials from '../../types/onyx/Credentials';
 
-let credentials;
-let authToken;
-let supportAuthToken;
-let currentUserEmail;
+let credentials: Credentials;
```

Comment:
```suggestion
let credentials: Credentials | undefined;
```

`credentials` is `undefined` before its value is set by `Onyx.connect`, we should consider this case.